style.css

```diff
@@ -240,3 +288,76 @@ textarea {
 #form button {
   margin-right: 211px;
 }
+
+@media (min-width: 768px) {
+  #intro {
+    background: url("./images/Header-bg.svg ") no-repeat;
+    background-color: white;
+    background-position: 100%;
+  }
+
+  #intro-content {
+    margin: auto;
+    width: 50%;
+  }
+
+  .project {
+    flex-grow: 1;
+    flex-direction: row;
+    gap: 1%;
+    justify-content: space-between;
+    height: auto;
+  }
+
+  #project-properties {
+    min-width: 10%;
+    padding-bottom: 0;
+  }
+
+  .snapshot,
+  .snapshot2 {
+    flex-grow: 3;
+    width: 50%;
+    height: 50%;
+  }
+
+  .snapshot2 {
+    order: 1;
+  }
+
+  #about-me {
+    height: 900px;
+  }
+
+  #right-side {
+    flex-shrink: 2;
+  }
+
+  #right-side p {
+    width: 300px;
+  }
+
+  #list {
+    margin-top: 200px;
+    margin-left: -40px;
+  }
```

Comment:
- [ ] Please add a line between frameworks and skills as implemented in  the Figma template. Also add one below skills 👍 
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/166071085-c141b4fc-19cd-46e8-ab2d-3cc2bc859726.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/166071143-86c95d2a-d57a-4bef-a738-47af7a485f96.png)

- [ ] You should add more space in between the languages and the down arrow. You should also add more space between the arrows and the Frameworks and skills. Note that they should match that of the Figma 🏄 
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/166071514-aba6c99b-f3e6-4d2e-9f39-d768824cdebe.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/166071585-b699d81e-f68b-4337-82bc-c7eac1db86e2.png)